### PR TITLE
Update CVE-2021-29101.json

### PR DESCRIPTION
**Pull request by RandallWilliams**
Esri allocates CVE-2021-29101

### CODE REVIEW

File: 2021/29xxx/CVE-2021-29101.json [hunk 1]
```diff
@@ -1,18 +1,100 @@
 {
```
Comment (cve-team) [inline 2021/29xxx/CVE-2021-29101.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/29xxx/CVE-2021-29101.json:1]: ('Extra data: line 100 column 1 (char 3316)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by RandallWilliams** [comment]
@cve-team please review and merge.

**Pull request merged**